rollup.config-es6.js

```diff
@@ -0,0 +1,55 @@
+const commonjs = require('rollup-plugin-commonjs');
+const nodeResolve = require('rollup-plugin-node-resolve');
+const replace = require('rollup-plugin-replace');
+const babel = require('rollup-plugin-babel');
+const uglify = require('rollup-plugin-uglify');
+const { minify } = require('uglify-es');
+
+const DEV = !!JSON.parse(process.env.BUILD_DEV || 'true');
+const DEMO = !!JSON.parse(process.env.BUILD_DEMO || 'false');
+
+const plugins = [
+  babel({
+    babelrc: false,
+    presets: [
+      [
+        'es2016',
+      ]
+    ],
+    plugins: [
+      'external-helpers',
```

Comment:
If we don't transpile, we don't need this.

Reply:
We need this if we do browser-env.

Reply:
Without it a warning is issued. Lets keep it.